catalog/models.py

```diff
@@ -418,40 +417,39 @@ class Meta:
     )
 
     slug = models.SlugField(
-        blank=False, unique=False, max_length=SLUG_MAX_LENGTH
+        blank=False, unique=False, max_length=SLUG_MAX_LENGTH,
     )
 
     def __str__(self):
         return self.name
 
-    # @todo #fidals/shopelectro#741:15m Make _generate_short_slug actual.
-    #  _generate_short_slug contains unrelevant code and docs.
-    def _generate_short_slug(self, max_length=SLUG_MAX_LENGTH) -> None:
+    def _slugify(self) -> None:
         """
-        Generate slug with limited length.
+        Make a slug from the name.
 
-        Slug is autogenerated from name. But name can be 2000 symbols in length.
-        This method regenerate slug to keep it length less then SLUG_MAX_LENGTH.
+        1. Translate the name's punctution chars to "_"
+        2. Slugify the name
```

Comment:
this method provides safe slug creating for a given tag. That's all, that client code should know about it. No realization details in an interface. Doc is the part of an interface, i believe.